lib/network.sh

```diff
@@ -149,15 +149,17 @@ else
   export BAREMETALV6_POOL_RANGE_END=""
 fi
 
+export REGISTRY_PORT=${REGISTRY_PORT:-"5000"}
```

Comment:
I understand why you would want to have it parameterize, however, if you go that way, it should either not be configurable by the user, or you should also parameterize the docker command that starts the registry https://github.com/metal3-io/metal3-dev-env/blob/master/02_configure_host.sh#L133

Reply:
Fixed. Thanks